services/motion/builtin/builtin.go

```diff
@@ -279,14 +279,14 @@ func (ms *builtIn) MoveOnGlobe(
 	heading float64,
 	movementSensorName resource.Name,
 	obstacles []*spatialmath.GeoObstacle,
-	motionCfg *motion.MotionConfiguration,
+	motionCfg *motion.Configuration,
```

Comment:
I got a lint warning saying that calling this type `motion.MotionConfiguration` is duplicative & to instead call it `motion.Configuration`

Reply:
I changed it back